esinstall/src/entrypoints.ts

```diff
@@ -275,7 +275,9 @@ function* forEachWildcardEntry(
   let valueDirectoryFullPath = path.join(cwd, valueDirectoryName);
 
   if (existsSync(valueDirectoryFullPath)) {
-    let filesInDirectory = readdirSync(valueDirectoryFullPath);
+    let filesInDirectory = readdirSync(valueDirectoryFullPath).filter(
+      (filepath) => statSync(path.join(valueDirectoryFullPath, filepath)).isFile(), // ignore directories
```

Comment:
This is the fix here; feel free to request an alternate approach if you’d do it differently! (the code in this file is so clean and I feel like this is a “messy” change)

Reply:
Looks fine to me!